src/app/helptext/task-calendar/replication/replication.ts

```diff
@@ -191,25 +191,25 @@ from the source dataset.'),
  <code>readonly</code> property during replication.'),
 
```

Comment:
Sync up any changes from the wizard help text into this file, or rework the strings to call help text from a single source file.